meetings/conferences/tests.py

```diff
@@ -1,38 +1,143 @@
-from django.test import TestCase
-from unittest import skip
+from django.test import TestCase, RequestFactory
+import factory
+from conferences.models import Conference
+from approvals import models as approvalModels
+from submissions import models as submissionModels
+from django.contrib.auth.models import User
+from permissions import ConferencePermissions
+from serializers import ConferenceSerializer
+from views import ConferenceViewSet
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = User
+
+
+class ConferenceFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Conference
+
+
+class ApprovalFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = approvalModels.Approval
+
+
+class SubmissionFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = submissionModels.Submission
+
+    # filling out NON NULL field
+    approval = factory.SubFactory(ApprovalFactory)
 
 
 class TestPermissions(TestCase):
-    @skip('use several tests to test different permissions aspects')
+    def setUp(self):
+        self.user = UserFactory(id=13)
+        self.conference = ConferenceFactory(
+            admin=self.user
+        )
+        self.request = RequestFactory().get('./fake_path')
+
     def test_conference_creator(self):
-        pass
+        self.view = ConferenceViewSet()
+        self.request.user = self.user
+        self.confPermissions = ConferencePermissions()
+        self.assertTrue(self.confPermissions.has_permission(self.request, self.view))
```

Comment:
I misunderstood the permissions file. Permissions tests will need further development